src/pages/HomePage.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+//made for testing with Router, can be deleted by whoever is doing the HomePage
+function Home(){
```

Comment:
nit: *`HomePage`? 

I'm not sure yet what we should call the page components. Having `Page` in the name seems already  redundant, especially given the `/pages` directory they live in.